examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -0,0 +1,88 @@
+"""
+========================================
+Release Highlights for scikit-learn 0.23
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.23, which comes
+with many bug fixes and new features! We detail below a few of the major
+features of this release. For an exhaustive list of all the changes, please
+refer to the :ref:`release notes <changes_0_23>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install scikit-learn
+"""
+
+##############################################################################
+# Generalized Linear Models
+# -------------------------
+# Long-awaited Generalized Linear Models with non-normal loss functions are now
+# available. In particular, three new regressors were implemented:
+# :class:`~sklearn.linear_model.PoissonRegressor`,
+# :class:`~sklearn.linear_model.GammaRegressor`, and
+# :class:`~sklearn.linear_model.TweedieRegressor`. The Poisson regressor can be
+# used to model positive integer counts, or relative frequencies. Read more in
+# the :ref:`User Guide <Generalized_linear_regression>`.
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import PoissonRegressor
+
+n_samples, n_features = 100, 20
+rng = np.random.RandomState(0)
+X = rng.randn(n_samples, n_features)
+y = ((10 + X[:, 5]) * 10).astype(int)  # easy target correlated with X[:, 5]
```

Comment:
@lorentzenchr any suggestions for a more compelling example, while still keeping it minimal?

same below about the poisson loss for GBDTs ;)

Thanks!

Reply:
```suggestion
y = rng.poisson(lam=2 + X[:, 5] / 10)  # easy target correlated with X[:, 5]
```
This gives at least a few `y=0`entries.

Reply:
This seems to lead to bad scores (-0.048). Ideally we would have something that shows the estimators work well ^^